File: eBolnica/UI/ebolnica_desktop/lib/models/Response/broj_zaposlenih_po_odjelu_response.dart
class BrojZaposlenihPoOdjeluResponse {
  int odjelId;
  String nazivOdjela;
  int ukupanBrojZaposlenih;

  BrojZaposlenihPoOdjeluResponse({
    required this.odjelId,
    required this.nazivOdjela,
    required this.ukupanBrojZaposlenih,
  });

  factory BrojZaposlenihPoOdjeluResponse.fromJson(Map<String, dynamic> json) {
    return BrojZaposlenihPoOdjeluResponse(
      odjelId: json['odjelId'],
      nazivOdjela: json['nazivOdjela'],
      ukupanBrojZaposlenih: json['ukupanBrojZaposlenih'],
    );
  }
}

File: eBolnica/eBolnica.Model/Response/BrojZaposlenihPoOdjeluResponse.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eBolnica.Model.Response
{
    public class BrojZaposlenihPoOdjeluResponse
    {
        public int OdjelId { get; set; }
        public string NazivOdjela { get; set; } = null!;
        public int UkupanBrojZaposlenih { get; set; }
    }
}
